elixir/apps/web/lib/web/live/actors/groups.ex

```diff
@@ -0,0 +1,247 @@
+defmodule Web.Actors.Groups do
+  use Web, :live_view
+  alias Domain.Actors
+
+  def mount(%{"id" => id}, _session, socket) do
+    with {:ok, actor} <-
+           Actors.fetch_actor_by_id(id, socket.assigns.subject,
+             preload: [:memberships],
+             filter: [
+               deleted?: false
+             ]
+           ) do
+      current_group_ids = Enum.map(actor.memberships, & &1.group_id)
+
+      socket =
+        socket
+        |> assign(
+          actor: actor,
+          current_group_ids: current_group_ids,
+          added: %{},
+          removed: %{}
+        )
+        |> assign_live_table("groups",
+          query_module: Actors.Group.Query,
+          sortable_fields: [],
+          hide_filters: [:provider_id],
+          limit: 5,
+          callback: &handle_groups_update!/2
+        )
+
+      {:ok, socket}
+    else
+      _other -> raise Web.LiveErrors.NotFoundError
+    end
+  end
+
+  def handle_params(params, uri, socket) do
+    socket = handle_live_tables_params(socket, params, uri)
+    {:noreply, socket}
+  end
+
+  def handle_groups_update!(socket, list_opts) do
+    with {:ok, groups, metadata} <-
+           Actors.list_editable_groups(socket.assigns.subject, list_opts) do
+      {:ok,
+       assign(socket,
+         groups: groups,
+         groups_metadata: metadata
+       )}
+    end
+  end
+
+  def render(assigns) do
+    ~H"""
+    <.breadcrumbs account={@account}>
+      <.breadcrumb path={~p"/#{@account}/actors"}>Actors</.breadcrumb>
+      <.breadcrumb path={~p"/#{@account}/actors/#{@actor}"}>
+        <%= @actor.name %>
+      </.breadcrumb>
+      <.breadcrumb path={~p"/#{@account}/actors/#{@actor}/groups"}>
+        Group Memberships
+      </.breadcrumb>
+    </.breadcrumbs>
+    <.section>
+      <:title>
+        Group Memberships: <%= @actor.name %>
+      </:title>
+      <:help>
+        Add or remove Groups for a given Actor.
```

Comment:
(nit) 
```suggestion
        Add or remove Groups for this User.
```

Reply:
Ah I see, I was under the impression Service Accounts can't be in groups

Reply:
Being in a group would be the only way to give them access to a resource right?

Reply:
Perhaps I could say:
```
Add or remove Groups for a given User / Service Account
```

Reply:
Actor seems appropriate since we kind of committed to it I think by now

Reply:
👍